src/Orleans.Clustering.ZooKeeper/Orleans.Clustering.ZooKeeper.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <AssemblyName>Orleans.Membership.ZooKeeper</AssemblyName>
+    <AssemblyName>Orleans.Clustering.ZooKeeper</AssemblyName>
     <RootNamespace>OrleansZooKeeperUtils</RootNamespace>
```

Comment:
here too. Why is it OrleansZooKeeperUtils, not Orleans.ZooKeeperClustering, or Orleans.Clustering.ZooKeeper. We should update root namespace too right?

Reply:
Because we decided to not to modify namespaces in existing assemblies.